src/mlpack/methods/softmax_regression/softmax_regression.hpp

```diff
@@ -71,8 +72,7 @@ class SoftmaxRegression
   SoftmaxRegression(const size_t inputSize = 0,
                     const size_t numClasses = 0,
                     const bool fitIntercept = false);
-
-  /**
+    /**
```

Comment:
We could provide another constructor that takes the callbacks as an additional argument, similar to the train method.